### ISSUE TITLE
ethanol is not in the chemistry tab

### ISSUE THREAD

**Issue by MACMAN2003** [space-wizards/space-station-14#26737]
## Description

ethanol doesn't show up in the chemistry tab in the guidebook
**Reproduction**

open the guidebook, go to chemistry tab, type in "ethanol"
**Screenshots**

**Additional context**

**Comment by Dutch-VanDerLinde** [comment]
its in the bartending tab

**Comment by MACMAN2003** [comment]
yeah, but ethanol is also important for chemistry so it'd make sense for it to be there as well, especially since typing "ethanol" in the bartending tab doesn't do anything

**Comment by Dutch-VanDerLinde** [comment]
yeah its bulldozed by all the other drinks because by typing in "ethanol" you're getting the drinks which have an effect that adds ethanol to the solution
